reposilite-site/data/guides/migration/artifactory.md

```diff
@@ -0,0 +1,38 @@
+---
+id: artifactory
+title: Artifactory
+---
+
+### Instructions
+
+1. Export Each Repository from Artifactory.
+    Be sure to check the `.m2 compatible export` box to get an export that reposilite can use. Also, we recommend excluding artifactory build metadata if your version of Artifactory has this feature.
+2. Move the resulting export files to your new reposilite server and extract them to the repository folder you would like them in.
+    Depending on the organization of your new repositories, you may need to move your artifacts out of parent folders to get them in the location you want them in your new repository.
+3. Configure the new repository in reposilite, if you haven't already, making sure it is loading files from the correct directory.
+    You should be able to browse the artifacts once configuration reloads.
```

Comment:
I'd end this section by adding the link to this guide: https://reposilite-nl595cx48-dzikoysk.vercel.app/guide/repositories below the 3rd paragraph